.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @hmcts/idam
```

Comment:
unsure if this is the right team, is this chart actually in use? looks to be done by the old family public law team

Reply:
Well it hasn't been updated in 3 years so I'd imagine it's not in use but I'll do a search to see if anything is consuming it. Any advice on a person or slack channel that can be contacted for info?

Reply:
see if its in use and if it isn't then archive it, check the contact channel for any team that is using it if you find it